src/mono/mono/mini/aot-compiler.c

```diff
@@ -14355,6 +14382,50 @@ add_interp_in_wrapper_for_sig (MonoAotCompile *acfg, MonoMethodSignature *sig)
 	add_method (acfg, wrapper);
 }
 
+static gboolean
+mono_aot_add_direct_pinvoke (MonoAotCompile *acfg, char *dpi)
```

Comment:
Static functions don't need mono_ or mono_aot prefix.

Reply:
Thanks, changed to `process_direct_pinvokes`